chia/wallet/wallet_node.py

```diff
@@ -175,20 +176,37 @@ def rollback_request_caches(self, reorg_height: int):
         for cache in self.untrusted_caches.values():
             cache.clear_after_height(reorg_height)
 
-    async def get_key_for_fingerprint(self, fingerprint: Optional[int]) -> Optional[PrivateKey]:
+    async def get_private_key(
```

Comment:
nit but the renaming seems to be unrelated to the PR's topic.

Reply:
I renamed it because the exisiting behavior and intended use case is broader than simply getting a key by fingerprint. If there were other callers, perhaps that would have influenced taking this in a different direction, but as currently used, it's really about attempting to fetch a specific master key for wallet syncing, and failing that, (optionally) returning the first private key